src/AmplifyMapLibreRequest.ts

```diff
@@ -117,13 +117,13 @@ export default class AmplifyMapLibreRequest {
       url = `https://maps.geo.${this.region}.amazonaws.com/maps/v0/maps/${url}/style-descriptor`;
     }
 
-    if (new URL(url).hostname.endsWith(".amazonaws.com")) {
```

Comment:
I can't comment on it, but the description above for `resourceType` 404's https://github.com/mapbox/carmen/blob/master/carmen-geojson.md

Reply:
yeah, I just saw that too.
Can we remove those?
Doesn't seem to work anymore.
https://maplibre.org/maplibre-gl-js-docs/api/properties/ only talks about return type but nothing about the `url` and `resourceType` arguments

Reply:
yeah, if there's no replacement it's better to remove